com.microsoft.teams.Teams.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "com.microsoft.teams.Teams",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "19.08",
+  "command": "teams",
+  "tags": [
+    "proprietary"
+  ],
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=all",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "ar",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t ${FLATPAK_DEST}/lib/",
+        "install -Dm0755 $(which ar) ${FLATPAK_DEST}/bin/"
+      ]
+    },
+    {
+      "name": "desktop-file-edit",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 $(which desktop-file-edit) ${FLATPAK_DEST}/bin/"
```

Comment:
Can you just put it into build commands of Teams?

Reply:
I prefer to keep it separated. For me compiling or installing a dependency from the SDK is the same and should be kept in its own module.

Reply:
It's not always about personal preferences but uniformity with existing applications.

Reply:
For the record, I strongly disagree with you.

But that's not worth the hassle. I'll just do it for the `ar` and `desktop-file-edit` modules, but not the shell scripts. Lot of existing applications also have them separated.